doc/customization_steps.rst

```diff
@@ -0,0 +1,55 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _customization_steps:
+
+#######################################
+3. General Guidelines for Modeling

```

Comment:
> This guide may be the first point of the overall list rather than the 3rd, what do you think?

I think we if put this section in the front, it will create more questions than answering them. It will require expanding this page to fill the gaps and explain everything, which defies its main purpose -- to be concise summary. Expanding it make it close to current `Customization basics` page. If we do not expand people may think what is `natura.tiff` and `cutouts`. I think a sole purpose of this section to give brief summary of steps, not introduction. What do you think?

Reply:
> In this section we shall reference the validation and ok to reference the custom_validation file and how to feed-in custom data for users that do have country-specific data

I have included the validation part as suggeted.

Reply:
As a refence, as discussed today, it may be good to revise this steps as a preliminary introduction for the overall full tutorial, while moving more advanced concepts into the relative steps.
Great Yerbol :D This seems the last step to finalize. Congratulations!

Reply:
I have made it as short summary of Model customization section for better navigation.